<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamDialog</class>
 <widget class="QDialog" name="TeamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbToevoegen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Toevoegen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbWijzigen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Wijzigen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbVerwijderen">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Verwijderen</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>380</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
